benches/full_solvency_flow.rs

```diff
@@ -69,7 +69,7 @@ fn generate_zk_proof_benchmark(_c: &mut Criterion) {
 
     let csv_file = format!("benches/csv/entry_2_{}.csv", LEVELS);
 
-    let assets_sum = [Fp::from(556863u64)]; // greater than liabilities sum (556862)
+    let assets_sum = [Fp::from(556863u64), Fp::from(556863u64)]; // greater than liabilities sum (556862)
```

Comment:
Actually benchmarking doesn't work now because the csv files are based on a single balance. 
We can solve that by updating the csv files on AWS or by making the library more modular such that you can specify the N_ASSETS and all the related functions such as:

- csv_parser
- poseidon_params and specs

are modified accordingly.